### PR TITLE
 [FIX] base.view_res_partner_filter: Add filter names

### PR DESCRIPTION
**Pull request by IOAyman**
## Current behavior before PR:
Some filters in `base.view_res_partner_filter` are missing `name` attribute, which makes them a little difficult to accurately reference using `xpath`.

## Desired behavior after PR is merged:
Easily reference all filters in `base.view_res_partner_filter`

***PS:** This affects the [11.0][0] branch as well, so it should be ported.*
- - -
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

[0]: https://github.com/odoo/odoo/blob/11a3d100c39a56f1cffc073b5f3c30e3978456de/odoo/addons/base/res/res_partner_view.xml#L400

### CODE REVIEW

**Comment by noureddineme** [comment]
LGTM :+1:

File: odoo/addons/base/res/res_partner_view.xml [hunk 1]
```diff
@@ -342,9 +343,9 @@
                    <field name="user_id"/>
                    <field name="parent_id" domain="[('is_company','=',1)]" operator="child_of"/>
                    <group expand="0" name="group_by" string="Group By">
-                       <filter name="salesperson" string="Salesperson" domain="[]" context="{'group_by' : 'user_id'}" />
-                       <filter string="Company" context="{'group_by': 'parent_id'}"/>
-                       <filter string="Country" context="{'group_by': 'country_id'}"/>
+                       <filter name="salesperson" string="Salesperson" context="{'group_by' : 'user_id'}" />
```
Comment (mart-e) [inline odoo/addons/base/res/res_partner_view.xml]: why do you change that line?

Comment (IOAyman) [reply odoo/addons/base/res/res_partner_view.xml]: This filter's supposed to be for grouping. The empty `domain="[]"` attribute is not necessary, is it?

Comment (mart-e) [reply odoo/addons/base/res/res_partner_view.xml]: always keep the diff minimal, please check our [stability guidelines](https://github.com/odoo/odoo/wiki/Contributing#what-does-stable-mean), we make no cleaning or improvement on stable versions

**Comment by mart-e** [comment]
Thanks, merged in 10.0

* 5f0b7942d55 [FIX] base: Add filter names
* d0e71fccf83 [CLA] IOAyman's signature

**Comment by IOAyman** [comment]
1st contribution to Odoo 🎆 Glad to help!

**Comment by Yenthe666** [comment]
Good job @IOAyman! Keep up the good work. 👍

**Comment by IOAyman** [comment]
Thanks for the support @Yenthe666 😄

**Pull request closed without merging**